server/internal/server/milvus_integration_test.go

```diff
@@ -0,0 +1,47 @@
+//go:build integration
+// +build integration
+
+package server
+
+import (
+	"context"
+	"testing"
+
+	"github.com/milvus-io/milvus-sdk-go/v2/client"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMilvusCreateListUpdateDeleteVectorStores(t *testing.T) {
```

Comment:
Could you add some comment and explain the setting that is required to run this test?

Reply:
Added in README